docs/pod-metrics.md

```diff
@@ -57,9 +57,9 @@ So to mimic the [logic](https://github.com/kubernetes/kubernetes/blob/v1.17.3/pk
 
 For example:
 
-* To get the list of pods that are in the `Unknown` state, you can run the following PromQL query: `sum(kube_pod_status_phase{phase="Unknown"}) by (namespace, pod) or (count(kube_pod_deletion_timestamp) by (namespace, pod) * sum(kube_pod_status_reason{reason="NodeLost"}) by(namespace, pod))`
+* To get the list of pods that are in the `Unknown` state, you can run the following PromQL query: `kube_pod_status_phase{phase="Unknown"} or count(kube_pod_deletion_timestamp) by (namespace, podname) * sum(kube_pod_status_reason{reason="NodeLost"}) by(namespace, podname)`
```

Comment:
Remove the `sum` part of `sum(kube_pod_status_phase{phase="Unknown"})` because the example says to get a list.

Reply:
I don't think we have a podname label. Do you have some relabeling configs which might be changing the label in your setup?

https://github.com/kubernetes/kube-state-metrics/blob/master/docs/pod-metrics.md

Reply:
Ah, right. I missed that. I found the replacement in our setup. Thank you.